add PID declaration docs (#293)
artefactual/archivematica-docs: user-manual/transfer/transfer.rst

   If you are creating bags, checksum files will be created as part of the
   bagging process. You do not need to create checksums manually.

.. _transfer-pids:

Transfers with existing persistent identifiers
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Archivematica can import persistent identifiers that were created

> outside of Archivematica. Perhaps your workflow includes DOI identifiers or

Reviewer: Would change this to be more neutral, i.e.:

```suggestion
outside of Archivematica, such as DOI identifiers or
```

identifiers created by your cataloging system. You can bind these to the
PREMIS objects created by Archivematica for your files by including an
identifiers.json file in the metadata sub-directory.

.. image:: images/identifiers.json.*
  :align: center
  :width: 50%
  :alt: A transfer containing an identifiers.json file

These persistent identifiers (PID) will be added during the Ingest Bind PIDs
microservice, regardless of whether the Bind PIDs microservice is enabled or
not.

.. image:: images/PID_declaration.*
  :align: center
  :width: 50%
  :alt: A persistent identifier (PID) is declared as a link to a PREMIS object

Once Ingest completes, the PID will be linked to the premis:objectIdentifier in
the AIP METS file.

.. image:: images/PID_linked_to_PREMIS.*
  :align: center
  :width: 50%

>   :alt: PIDs linked to PREMIS objects

Reviewer: Any code examples should be included as text, rather than screenshots, so that they are more accessible (i.e. text can be zoomed as big as needed, whereas images get blurry; text can also be read by a screenreader.) This is something I've been slowly changing over time so no need to do it now if you don't have time, though.

For an example, see the METS examples in import-metadata.rst
